frontend/src/features/entity_selector/EntitySelectButton.tsx

```diff
@@ -180,29 +184,44 @@ const VideoInput = ({
           width: variant === 'full' ? '100%' : 'auto',
         }}
       >
-        <Button
-          fullWidth={variant === 'full' ? true : false}
-          onClick={toggleSuggestions}
-          size="small"
-          variant="contained"
-          color="secondary"
-          disabled={disabled}
-          sx={
-            variant === 'full'
-              ? { minHeight: '100px', fontSize: '1rem' }
-              : { minWidth: '80px', fontSize: { xs: '0.7rem', sm: '0.8rem' } }
-          }
-          disableElevation
-          data-testid={`entity-select-button-${variant}`}
-        >
-          {variant === 'full'
-            ? t('entitySelector.selectAVideo')
-            : t('entitySelector.select')}
-        </Button>
+        {smallScreen && variant === 'compact' ? (
+          <IconButton
+            onClick={toggleSuggestions}
+            size="small"
+            color="secondary"
+            disabled={disabled}
+            sx={{ fontSize: { xs: '0.7rem', sm: '0.8rem' } }}
+            data-testid={`entity-select-button-${variant}`}
+          >
+            <Search />
```

Comment:
The search icon is temporary, and will be replaces by:
- the swipe gesture
- swipe up / down visual indications
- a mention in the tutorial

In case this temporary icon is not satisfying enough, here are some alternatives

| ![capture](https://github.com/tournesol-app/tournesol/assets/39056254/083a9573-316e-4afe-a1b4-abeae494a919) | ![capture](https://github.com/tournesol-app/tournesol/assets/39056254/fc6c5753-b714-4ac1-8cf4-0d58013a22fc) | ![capture3](https://github.com/tournesol-app/tournesol/assets/39056254/649dc4c4-149a-46d4-bdec-57df1260c855) |
|---|---|---|

Reply:
the search icon is indeed the best option, I think